functions/log/index.js

```diff
@@ -49,52 +49,6 @@ function getLogEntries() {
 }
 // [END functions_log_retrieve]
 
-// [START functions_log_get_metrics]
-// By default, the client will authenticate using the service account file
-// specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
-// the project specified by the GCLOUD_PROJECT environment variable. See
-// https://github.com/googleapis/google-cloud-node/blob/master/docs/authentication.md
-const Monitoring = require('@google-cloud/monitoring');
```

Comment:
@stew-r @labtopia I take it it's not worth adding this (back?) to [this page](https://cloud.google.com/functions/docs/monitoring/metrics) in a "Viewing metrics programmatically" section?

Reply:
This doesn't look like a sample from the Cloud Functions docs?

Reply:
So we can safely delete it?

Reply:
Stewart is busy, so I'll defer to @labtopia on this. (Happy to remove it if it's not worth adding back to the docs.)

Reply:
We just got a bug about how useless the metrics doc is. Anything that provides a little more information is worth putting in. Please file a bug with the sample link and assign it to me, and I'll put it in.

Reply:
On second thought: it seems a bit duplicative of [this sample](https://github.com/googleapis/nodejs-monitoring/blob/master/samples/metrics.js#L210).

@labtopia should we use that instead?

Reply:
This is the comment in the bug we recently received for monitoring:

"This is useless. How do I turn on metrics for Cloud Functions, then how do I access them, and show a simple example."

At least this user seems to want to see an example of how to get it to work for GCF specifically.

Reply:
Add the sample back in and opened b/134742003